oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/RelatedSelectField/RelatedSelectFieldInternal.jsx

```diff
@@ -150,7 +150,13 @@ export class RelatedSelectFieldInternal extends RemoteSelectField {
         <SelectField
           allowAdditions={this.error ? false : uiProps.allowAdditions}
           fieldPath={compProps.fieldPath}
-          options={this.state.suggestions}
+          options={_uniqBy(
+            [
+              ...serializeSuggestions(initialSuggestions),
```

Comment:
Is this to support providing suggestions from outside of component (e.g. Terka's issue?)